src/core/Akka.Streams.Tests.TCK/Akka.Streams.Tests.TCK.csproj

```diff
@@ -18,8 +18,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(TestSdkVersion)" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
     <PackageReference Include="Reactive.Streams.TCK" Version="1.0.2" />
-    <PackageReference Include="NUnit" Version="3.12.0" />
+    <!-- !!!WARNING!!! ALWAYS MATCH NUNIT VERSION WITH THE VERSION USED BY REACTIVE TCK !!!WARNING!!! -->
+    <PackageReference Include="NUnit" Version="3.7.1" />
```

Comment:
If we don't explicitly include an `NUnit` reference here, we'll take whatever `NUnit` version is used by `Reactive.Streams.TCK` as a transitive dependency. So maybe the safest way to handle this is to not explicitly specify an NUnit dependency at all? What do you think?

Reply:
Good idea, I'll do a quick check and update.